egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -326,12 +326,12 @@ def __init__(
 
     def _reset_parameters(self, initial_speed: float):
         std = 0.1 / initial_speed
-        a = (3**0.5) * std
+        a = (3 ** 0.5) * std
         nn.init.uniform_(self.weight, -a, a)
         if self.bias is not None:
             nn.init.constant_(self.bias, 0.0)
         fan_in = self.weight.shape[1] * self.weight[0][0].numel()
-        scale = fan_in**-0.5  # 1/sqrt(fan_in)
+        scale = fan_in ** -0.5  # 1/sqrt(fan_in)
```

Comment:
The reset_parameters in torch rnn.py looks like:
```
    def reset_parameters(self) -> None:
        stdv = 1.0 / math.sqrt(self.hidden_size)
        for weight in self.parameters():
            init.uniform_(weight, -stdv, stdv)
```
so I suggest to set: 
```
     scale = self.hidden_size ** -0.5
```

Reply:
OK. Thanks.